### PR TITLE
feat: link TOC from presentation index

### LINKED ISSUE #53: Presentation
Tried using: https://skade.github.io/rust-three-days-course/presentation/index.html
getting the same results as running locally.

expected: index page that lets me navigate the presentation/index the contents
got: index page that lets select a language but not going through the presentation

### PR DESCRIPTION
**Pull request by emig**
- moved TOC related files into the presentation folder tree, under toc folder
- linked TOC from each language `intro.chapter` page
- link the presentation's index on the GH page on the README
- updated the task to generate html from toc md files and run the task

closes #53

### CODE REVIEW

File: presentation/chapters/en-US/intro.chapter [hunk 1]
```diff
@@ -1 +1,3 @@
-# Rust in three days
\ No newline at end of file
+# Rust in three days
+
+[TOC](toc/english.html)
```
Comment (ericho) [inline presentation/chapters/en-US/intro.chapter]: Maybe the TOC acronym can be expanded.

**Comment by skade** [comment]
Would it also make sense to just render them through the Javascript in that case? Currently, they are hand-compiled.

**Comment by emig** [comment]
Sure, I would do that in a later PR

**Comment by emig** [comment]
Tried with a local server and worked. It seems I can't navigate this on github pages, being a fork...

File: presentation/chapters/en-US/intro.chapter (hunk 1 above)
Comment (emig) [inline presentation/chapters/en-US/intro.chapter]: Ah, good idea, will expand that and also I18n the acronym

**Comment by emig** [comment]
@ericho had to push the branch to master on my fork, to get this rendered, but payed off:

https://emig.github.io/rust-three-days-course/presentation/#/

you can check the end result.

**Comment by Hoverbear** [comment]
This may allow for the deprecation of https://github.com/skade/rust-three-days-course/blob/master/src/bin/build-indices.rs

**Comment by ericho** [comment]
@emig I verified the result in your fork. LGTM.

**Comment by emig** [comment]
@Hoverbear added an issue for removing the `build-indices task` https://github.com/skade/rust-three-days-course/issues/55

**Pull request merged**